examples/minimal/README.md

```diff
@@ -0,0 +1,18 @@
+# `@shopify/checkout-ui-react` minimal example
+
+```sh
+yarn
+```
+
+## Run dev server
```

Comment:
I know it's logged at the beginning of `yarn start`, but maybe we should add where the app is served to the readme? (http://localhost:8080/)